### PR TITLE
Add rendering for landuse=salt_pond

### LINKED ISSUE #545: Add rendering for landuse=salt_pond
The following landcover types are currently not rendered:
- ~~landuse=harbour, see also https://trac.openstreetmap.org/ticket/3612~~ rejected in https://github.com/gravitystorm/openstreetmap-carto/issues/545#issuecomment-277207366 and https://github.com/gravitystorm/openstreetmap-carto/issues/545#issuecomment-341894371
- landuse=salt_pond, see also https://trac.openstreetmap.org/ticket/4470
- ~~natural=dune~~, see also https://trac.openstreetmap.org/ticket/3815
- ~~natural=fell~~ rejected in https://github.com/gravitystorm/openstreetmap-carto/issues/545#issuecomment-98479426 and https://github.com/gravitystorm/openstreetmap-carto/issues/545#issuecomment-341895192 , see also https://trac.openstreetmap.org/ticket/3033
- ~~and natural=scree~~ (done), see also https://trac.openstreetmap.org/ticket/3033
- ~~natural=reef, see also https://trac.openstreetmap.org/ticket/4416~~ done

It would be nice to have these tags rendered.

### PR DESCRIPTION
**Pull request by jeisenbe**
Fixes #545
Follow-up to #3905

Changes proposed in this pull request:
- Add rendering of [landuse=salt_pond](https://wiki.openstreetmap.org/wiki/Tag:landuse=salt_pond) areas with fill in ocean color (slightly lighter than water color,  `#b5d7e3`  - Lch(84,13,227)
- Add pattern overlay for landuse=salt_pond from @imagico alt-colors style
- Add text label for landuse=salt_pond
Test rendering with links to the example places:

https://www.openstreetmap.org/#map=15/41.5849/9.2909
Salt evaporation pond area in Corsica

Before
salt-pond-before-15:41 5849:9 2909

After
salt-pond-after-water-color-15

https://www.openstreetmap.org/#map=14/-34.8186/138.5977
Salt evaporation ponds (salterns) in South Australia

Before
salt-pond-before-australia-14:-34 8186:138 5977

After
salt-pond-after-australia-14:-34 8186:138 5977

### CODE REVIEW

**Comment by jeisenbe** [comment]
It's supposed to be a subtle difference. But since it makes sense in the context of changing the ocean-color too, I have removed that change and just used `@water-color` for `salt_pond`, for now.

File: project.mml [hunk 1]
```diff
@@ -1464,10 +1465,10 @@ Layer:
...
                                                     'residential', 'garages', 'meadow', 'grass', 'allotments', 'forest', 'farmyard', 'farmland',
                                                     'greenhouse_horticulture', 'retail', 'industrial', 'railway', 'commercial', 'brownfield', 'landfill',
-                                                    'construction', 'military', 'plant_nursery') THEN landuse END,
+                                                    'construction', 'salt_pond', 'military', 'plant_nursery') THEN landuse END,
                 'natural_' || CASE WHEN "natural" IN ('peak', 'volcano', 'saddle', 'cave_entrance') AND way_area IS NULL THEN "natural" END,
-                'natural_' || CASE WHEN "natural" IN ('wood', 'water', 'mud', 'wetland', 'bay', 'spring', 'scree', 'shingle', 'bare_rock', 'sand', 'heath',
-                                                      'grassland', 'scrub', 'beach', 'glacier', 'tree', 'strait', 'cape')
+                'natural_' || CASE WHEN "natural" IN ('wood', 'peak', 'volcano', 'saddle', 'cave_entrance', 'water', 'mud', 'wetland', 'bay', 'spring',
```
Comment (imagico) [inline project.mml:1470]: That change does not seem to be related to this PR.

Comment (jeisenbe) [reply project.mml]: Oops, thank you for catching that.

**Comment by jeisenbe** [comment]
I think that the water color should evenually be changed to be the same as salt water lakes, and that should be a slightly lighter color, so the pattern will not look quite as strong then.
